skll/data/readers.py

```diff
@@ -695,19 +696,24 @@ def safe_float(text, replace_dict=None):
                          floats. Anything not in the mapping will be kept the
                          same.
     :type replace_dict: dict from str to str
+    :param text: The Logger instance to use to log messages. Used instead of
```

Comment:
The documentation here and below should have `logger` instead of `text` as the parameter name.